doc/frequent-itemsets.rst

```diff
@@ -0,0 +1,62 @@
+Frequent Itemsets
+=================
+
+.. figure:: icons/frequent-itemsets.png
+
+Finds frequent itemsets in the data.
+
+Signals
+-------
+
+#### Inputs:
+
+-  **Data**
+
+Data set
```

Comment:
Re spaces, the follow up paragraphs should be aligned with the start of the list item, which is the first non-whitespace character following the bullet.